C/other/ternarySearch.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h> 

```

Comment:
Better code has comments in it, as this method used recursion, try to include some comments as well, that will make it easier for a beginner to understand.